lib/consyncful/sync.rb

```diff
@@ -4,6 +4,7 @@
 require 'consyncful/item_mapper'
 require 'consyncful/persisted_item'
 require 'consyncful/stats'
+require 'consyncful/configuration'
```

Comment:
Same here. If it doesn't work. That's all good.

Reply:
I think it does! tests are still passing :) I've removed from both base and sync